code/src/Component/Header.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import  '../Style/Header.css';
+
+const Header = () => {
+  return (
+  <div className="header">
+    <h1 className="header-title">
+      <div className="header-title-row-1">
+        <span className="header-title-1">Happy</span> <span className="header-title-2">Thought</span>
```

Comment:
By just looking at the code here it was difficult to understand why you've used span inside h1. When looking at your site it is quite clear - but consider using some self-explaining class names.